node-backend/app/controllers/orgnizationController.js

```diff
@@ -0,0 +1,165 @@
+const db = require("../models/sequelize");
+const Organizations = db.organizations;
+const yup = require("yup");
+
+const schema = yup.object().shape({
+  name: yup.string().required(),
+  contact_email: yup.string().email(),
+  domain_name: yup.string().required(),
```

Comment:
how it satisfies valid domain validation?

Reply:
can you examples of domain name ?